core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/danalysis/CapellaAnalysisSelectorProvider.java

```diff
@@ -33,4 +33,10 @@ public boolean provides(DAnalysisSession session) {
     }
     return false;
   }
+  
+  @Override
+  public int getPriority() {
+    // Must be a higher priority than SmartDialogAnalysisSelectorProvider.
+    return 2;
```

Comment:
can you use SmartDialogAnalysisSelectorProvider.getPriority() + 1
or write a junit ensuring this requirement
thanks